client/src/App.tsx

```diff
@@ -4,23 +4,35 @@ import {
     Home,
     Profile
 } from "./pages"
+import SignUpForm, { SignUpInformation } from "./pages/SignUpForm"
```

Comment:
Did you mean to do something like `import SignUpForm from "./components/SignUp"`? I am getting a `can't find module` error here

Reply:
Will revert changes to old App.tsx version

Reply:
Done.